app/main/routes_frontend.py

```diff
@@ -648,7 +648,7 @@ def load_brew_sessions(uid=None):
         files = list(brew_archive_sessions_path().glob("[^_.]*#{}*.json".format(uid)))
     else:
         files = list(brew_archive_sessions_path().glob(file_glob_pattern))
-    brew_sessions = [parse_brew_session(file) for file in files]
+    brew_sessions = [parse_brew_session(file) for file in sorted(files, reverse=True)]
```

Comment:
Another improvement that needs to be made to the brew_sessions (doesn't need to be made here) is splitting the multiple machines one has so it is easier to locate the split between them. And or just intermingle all the sessions cause do I really need to scroll past my first alphabetical machine (ie. MachineA) to see the most recent session from say MachineB?

Thanks for ordering these in reverse order, that is great and puts recent on top.

Reply:
Thanks. Since all the rows can be moved the first three rows aren't completely locked, but this helps some.

I wondered about splitting the lists by machine too, but sorting by date seemed a bit more useful. More sorting options can be added too.